fiovb/host/main.c

```diff
@@ -115,6 +116,43 @@ static int write_persistent_value(const char *name, const char *value)
 	return -1;
 }
 
+static int delete_persistent_value(const char *name)
+{
+	char req1[MAX_BUFFER] = { '\0' };
+
+	struct fiovb_ctx ctx;
+	TEEC_Operation op;
+	TEEC_Result res;
+	uint32_t origin;
+
+	if (strlen(name) >= sizeof(req1)) {
+		fprintf(stderr, "Name value too large (should be up to %d)\n",
+				sizeof(req1));
+		return -1;
+	}
+
+	prepare_tee_session(&ctx);
+
+	strcpy(req1, name);
+	memset(&op, 0, sizeof(op));
+	op.paramTypes = TEEC_PARAM_TYPES(TEEC_MEMREF_TEMP_INPUT,
+					 TEEC_NONE, TEEC_NONE, TEEC_NONE);
+	op.params[0].tmpref.buffer = req1;
+	op.params[0].tmpref.size = strlen(req1) + 1;
+
+	res = TEEC_InvokeCommand(&ctx.sess, TA_FIOVB_CMD_DELETE_PERSIST_VALUE,
+				 &op, &origin);
+
+	terminate_tee_session(&ctx);
+
+	if (res == TEEC_SUCCESS)
+		return 0;
+
+	fprintf(stderr, "DELETE_PVALUE failed: 0x%x / %u\n", res, origin);
+
+	return -1;
```

Comment:
maybe better use POSIX error code define instead of integer literal?

Reply:
Done.